sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -133,6 +133,47 @@ export interface SearchAvailablePhoneNumbersRequest extends PhoneNumberSearchReq
     countryCode: string;
 }
 
+// @public
+export class SipRoutingClient {
+    constructor(connectionString: string, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: TokenCredential, options?: SipRoutingClientOptions);
+    deleteTrunk(fqdn: string, options?: OperationOptions): Promise<void>;
+    getTrunk(fqdn: string, options?: OperationOptions): Promise<SipTrunk>;
+    listRoutes(options?: OperationOptions): Promise<SipTrunkRoute[]>;
+    listTrunks(options?: OperationOptions): Promise<SipTrunk[]>;
```

Comment:
Our convetion is that `list*()` methods return paged async iterator.  It's better to use `getRoutes()` and `getTrunks()` for when returning arrays.

Reply:
Renamed 
listTrunks to getTrunks
listRoutes to getRoutes